docs/kb/semgrep-ci/ci-vs-cli.md

```diff
@@ -0,0 +1,71 @@
+---
+description: How to align your scan results between CI and CLI and understand differences in behavior.
+tags:
+  - Semgrep CI
+  - Semgrep CLI
+---
+
+# Semgrep in CI vs CLI: align your scan results and understand differences
```

Comment:
```suggestion
# Semgrep in CI vs CLI: align your SAST scan results and understand differences
```
Just a minor suggestion, feel free to ignore. We are going to create new sections in the docs for SSC in CLI, so we may want to make it clear this is for SAST findings only.

Reply:
yeah that's probably good - many of the configuration options apply to both but I didn't write it in a super inclusive way, and we don't have the same CLI options for SSC (yet).